joomla/joomla-cms
# [com_content] Repositioning Alternate Layout in article options and rename it

Rename Alternate Layout to Layout
Move it to the Main Article Options

Pull Request for Issue #10743  .

### Summary of Changes

### Testing Instructions
Select an Layout in your Article Options (Page 1 below languages) and check if it still works.

![image](https://cloud.githubusercontent.com/assets/828371/22471440/9e5a06a4-e7d2-11e6-9b96-8d95fad8daca.png)

### Documentation Changes Required
yes, new Screenshot and location description of the field
